### PR TITLE
SP: add markup for 1920 screens

### PR DESCRIPTION
**Pull request by AnnaRokalo**

### CODE REVIEW

File: src/shared/components/app/Costs/Costs.tsx [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
     <GridContainer fullBleed>
-      <Stack space={12} css={{ paddingTop: "$16" }}>
+      <Stack space={{ "@initial": 12, "@sm": 16 }} css={{ paddingTop: "$16" }}>
         {costs.map((cost) => {
           return (
             <Flex justify="between" key={cost.name}>
-              <Copy scale={8} color="neutrals-7">
+              <Copy
+                scale={{ "@initial": 9, "@sm": 8 }}
+                color={isSmallAndAbove ? "system-black" : "neutrals-7"}
```
Comment (VladKarchenia) [inline src/shared/components/app/Costs/Costs.tsx]: color={{ "@initial": "neutrals-7", "@sm": "system-black" }}

File: src/tracking/components/ShipmentCostsHeader/ShipmentCostsHeader.tsx [hunk 2]
```diff
@@ -14,21 +16,29 @@ export const ShipmentCostsHeader = ({
...
 }: IShipmentCostProps) => {
+  const isSmallAndAbove = useMedia([mediaQueries.sm], [true], false)
+
   return (
     <GridContainer fullBleed>
       <Stack space={8}>
         <Flex align="center" justify="between">
-          <Copy scale={10} color="neutrals-7" bold>
+          <Copy
+            scale={isSmallAndAbove ? 7 : 11}
```
Comment (VladKarchenia) [inline src/tracking/components/ShipmentCostsHeader/ShipmentCostsHeader.tsx]: use {{ "@initial": "", "@sm": "" }} for both **scale** and **color**

File: src/tracking/components/ShipmentCostsHeader/ShipmentCostsHeader.tsx [hunk 3]
```diff
@@ -14,21 +16,29 @@ export const ShipmentCostsHeader = ({
...
+            bold
+          >
             {title}
           </Copy>
-          <ButtonIcon
-            ariaLabel="Show details"
-            icon={showCostsDetails ? <IconChevronTop /> : <IconChevronDown />}
-            onClick={handleShowDetailsClick}
-          />
+          {isSmallAndAbove ? null : (
```
Comment (VladKarchenia) [inline src/tracking/components/ShipmentCostsHeader/ShipmentCostsHeader.tsx]: use `<Hidden above="sm">content</Hidden>` instead

File: src/tracking/components/ShipmentCosts/ShipmentCosts.tsx [hunk 4]
```diff
@@ -24,7 +27,12 @@ export const ShipmentCosts = ({ title, price, costs }: IShipmentCostsProps) => {
         showCostsDetails={showCostsDetails}
         handleShowDetailsClick={handleShowDetailsClick}
       />
-      {showCostsDetails ? <Costs costs={costs} /> : null}
+      {isSmallAndAbove ? (
```
Comment (VladKarchenia) [inline src/tracking/components/ShipmentCosts/ShipmentCosts.tsx]: "isSmallAndAbove || showCostsDetails" condition?

File: src/tracking/components/ShipmentRoutePoint/ShipmentRoutePoint.tsx [hunk 5]
```diff
@@ -32,11 +32,14 @@ export const ShipmentRoutePoint = ({
...
-        <Copy scale={8} color={isStepCompleted || isStepInProgress ? "system-black" : "neutrals-5"}>
+        <Copy
+          scale={{ "@initial": 8, "@sm": 8 }}
+          color={isStepCompleted || isStepInProgress ? "system-black" : "neutrals-5"}
+        >
           {status}
         </Copy>
         {date ? (
-          <Copy scale={9} color="neutrals-5">
+          <Copy scale={{ "@initial": 9, "@sm": 9 }} color="neutrals-5">
```
Comment (VladKarchenia) [inline src/tracking/components/ShipmentRoutePoint/ShipmentRoutePoint.tsx]: double scale 9 value

File: src/tracking/components/TrackingContainer/TrackingContainer.tsx [hunk 6]
```diff
@@ -109,219 +90,50 @@ export const TrackingContainer = () => {
...
-                  <Copy as="span" scale={8} color="system-white" bold>
-                    Create a shipment
-                  </Copy>
-                </Button>
-              </>
-            ) : null}
-          </GridContainer>
-        </Stack>
+      <GridContainer css={{ paddingBottom: "$48" }}>
+        <HeaderBar title="Shipment details" onClick={() => navigate("/")} />
```
Comment (VladKarchenia) [inline src/tracking/components/TrackingContainer/TrackingContainer.tsx]: "Quote details"?

File: src/tracking/components/TrackingContainer/TrackingContainer.tsx [hunk 7]
```diff
@@ -109,219 +90,50 @@ export const TrackingContainer = () => {
...
+        <Spacer size={{ "@initial": 16, "@sm": 24 }} />
+        <QuoteDetails shippingType={data.shippingType} status={data.status} />
       </GridContainer>
     )
-  }
-  return (
-    <GridContainer fullBleed css={{ paddingBottom: "$48" }}>
-      <HeaderBar title="Shipment details" onClick={() => navigate("/")} />
-      <Stack space={16}>
+  } else if (role === Role.Admin) {
```
Comment (VladKarchenia) [inline src/tracking/components/TrackingContainer/TrackingContainer.tsx]: split this else if into separate if's blocks

File: src/tracking/components/TrackingContainer/TrackingContainer.tsx [hunk 8]
```diff
@@ -109,219 +90,50 @@ export const TrackingContainer = () => {
...
-          </GridContainer>
-        ) : null}
-      </Stack>
-    </GridContainer>
-  )
+        <Spacer size={{ "@initial": 16, "@sm": 24 }} />
+        <ShipmentDetails />
+      </GridContainer>
+    )
+  } else
```
Comment (VladKarchenia) [inline src/tracking/components/TrackingContainer/TrackingContainer.tsx]: split as above

File: src/tracking/components/TrackingContainer/TrackingContainer.tsx [hunk 9]
```diff
@@ -109,219 +90,50 @@ export const TrackingContainer = () => {
...
+    return (
+      <GridContainer css={{ paddingBottom: "$48" }}>
```
Comment (VladKarchenia) [inline src/tracking/components/TrackingContainer/TrackingContainer.tsx]: you have all 3 blocks with the similar structure: GridContainer, HeaderBar, TrackingHeader, Spacer. So you could create a generic component and pass `<QuoteDetails shippingType={data.shippingType} status={data.status} />`, `<ShipmentDetails />` or `<ShipmentDetailsUnauthorized />` as a children

File: src/tracking/components/TrackingDetailsItem/TrackingDetailsItem.tsx [hunk 10]
```diff
@@ -1,19 +1,25 @@
 import { Spaces } from "@/config/theme/spacing"
-import { Copy, Spacer } from "@/shared/components"
+import { Copy, Spacer, TypographyScale } from "@/shared/components"
+import { Colors } from "@/config/theme/types"
+import { ResponsiveProp } from "@/utils"
 
 interface ITrackingDetailsItemProps {
   title: string
   titleIndent?: Spaces
+  titleColor?: Colors
```
Comment (VladKarchenia) [inline src/tracking/components/TrackingDetailsItem/TrackingDetailsItem.tsx]: I'll double check this types

File: src/tracking/components/QuoteDetails/QuoteDetails.tsx [hunk 11]
```diff
@@ -0,0 +1,115 @@
+import { useShipmentStateContext } from "@/shared/state"
```
Comment (VladKarchenia) [inline src/tracking/components/QuoteDetails/QuoteDetails.tsx]: imports order

File: src/tracking/components/QuoteDetails/QuoteDetails.tsx [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+    <GridContainer
+      fullBleed
+      css={{
+        "@initial": {
+          maxWidth: "100%",
+          paddingBottom: "$48",
+        },
+        "@sm": {
+          maxWidth: "565px",
+          paddingBottom: "$48",
```
Comment (VladKarchenia) [inline src/tracking/components/QuoteDetails/QuoteDetails.tsx]: paddingBottom is already set in initial point

File: src/tracking/components/ShipmentDetails/ShipmentDetails.tsx [hunk 13]
```diff
@@ -0,0 +1,273 @@
+import { useShipmentStateContext } from "@/shared/state"
```
Comment (VladKarchenia) [inline src/tracking/components/ShipmentDetails/ShipmentDetails.tsx]: imports order

File: src/tracking/components/ShipmentDetails/ShipmentDetails.tsx [hunk 14]
```diff
@@ -0,0 +1,273 @@
...
+  const isSmallAndAbove = useMedia([mediaQueries.sm], [true], false)
+
+  return (
+    <STrackingGrid
+      columns={{ "@initial": "1fr", "@sm": "1fr 1fr 1fr 1fr" }}
+      gap={{ "@initial": 16, "@sm": 24 }}
+      rows={"auto auto"}
+    >
+      <STrackingGridItem css={{ gridArea: "main" }}>
+        {isSmallAndAbove ? (
```
Comment (VladKarchenia) [inline src/tracking/components/ShipmentDetails/ShipmentDetails.tsx]: `<Hidden>`

File: src/tracking/components/ShipmentDetails/ShipmentDetails.tsx [hunk 15]
```diff
@@ -0,0 +1,273 @@
...
+          </Stack>
+        </STrackingGridItem>
+      ) : null}
+
+      <STrackingGridItem css={{ gridArea: "costs" }}>
+        <ShipmentCosts title="Costs" price={rate.price} costs={costs} />
+      </STrackingGridItem>
+
+      <STrackingGridItem css={{ gridArea: "labels" }}>
+        {isSmallAndAbove ? (
```
Comment (VladKarchenia) [inline src/tracking/components/ShipmentDetails/ShipmentDetails.tsx]: need to double check whether we should use <Hidden> or isSmallAndAbove

File: src/tracking/components/ShipmentDetailsUnauthorized/ShipmentDetailsUnauthorized.tsx [hunk 16]
```diff
@@ -0,0 +1,129 @@
+import { useShipmentStateContext } from "@/shared/state"
```
Comment (VladKarchenia) [inline src/tracking/components/ShipmentDetailsUnauthorized/ShipmentDetailsUnauthorized.tsx]: order

File: src/shared/components/app/Costs/Costs.tsx (hunk 1 above)
Comment (AnnaRokalo) [reply src/shared/components/app/Costs/Costs.tsx]: done

File: src/tracking/components/ShipmentCostsHeader/ShipmentCostsHeader.tsx (hunk 2 above)
Comment (AnnaRokalo) [reply src/tracking/components/ShipmentCostsHeader/ShipmentCostsHeader.tsx]: done

File: src/tracking/components/ShipmentCostsHeader/ShipmentCostsHeader.tsx (hunk 3 above)
Comment (AnnaRokalo) [reply src/tracking/components/ShipmentCostsHeader/ShipmentCostsHeader.tsx]: done

File: src/tracking/components/ShipmentCosts/ShipmentCosts.tsx (hunk 4 above)
Comment (AnnaRokalo) [reply src/tracking/components/ShipmentCosts/ShipmentCosts.tsx]: done

File: src/tracking/components/ShipmentRoutePoint/ShipmentRoutePoint.tsx (hunk 5 above)
Comment (AnnaRokalo) [reply src/tracking/components/ShipmentRoutePoint/ShipmentRoutePoint.tsx]: done

File: src/tracking/components/TrackingContainer/TrackingContainer.tsx (hunk 6 above)
Comment (AnnaRokalo) [reply src/tracking/components/TrackingContainer/TrackingContainer.tsx]: done

File: src/tracking/components/QuoteDetails/QuoteDetails.tsx (hunk 12 above)
Comment (AnnaRokalo) [reply src/tracking/components/QuoteDetails/QuoteDetails.tsx]: done

File: src/tracking/components/ShipmentDetails/ShipmentDetails.tsx (hunk 13 above)
Comment (AnnaRokalo) [reply src/tracking/components/ShipmentDetails/ShipmentDetails.tsx]: done

File: src/tracking/components/ShipmentDetailsUnauthorized/ShipmentDetailsUnauthorized.tsx (hunk 16 above)
Comment (AnnaRokalo) [reply src/tracking/components/ShipmentDetailsUnauthorized/ShipmentDetailsUnauthorized.tsx]: Done

**Pull request merged**